@kiva/kv-components/vue/stories/KvSideSheet.stories.js

```diff
@@ -45,7 +52,33 @@ const Template = (args, {
 		) {
 			console.log(category, action, label, property, value);
 		},
+		openModal(event) {
+			if (this.expandEffect) {
+				const card = event.currentTarget;
+				if (card) {
+					const rect = card.getBoundingClientRect();
```

Comment:
I think it would make sense to have the component itself do this logic, otherwise we'd need to find the element location anytime we use this type of animation. You could add a single property, like `animationSourceElement`, and use it in place of the two properties the component currently has.